Python/fitnesse/html_util.py

```diff
@@ -2,41 +2,41 @@
 
 from fitnesse.context import SuiteResponder, PageCrawlerImpl, PageData, PathParser, WikiPage, WikiPagePath
 
-
+class TestType:
+    SETUP: "SetUp"
+    TEARDOWN: "TearDown"
 class HtmlUtil:
 
+    @staticmethod
+    def get_Path(page_name: str, wiki_page: WikiPage):
```

Comment:
`get_Path` seems to be private. Consider putting it beyond the publicly callable function and indicate the intended access-level by an underscore, i.e., `_get_path`.